PendingReleaseNotes.md

```diff
@@ -26,6 +28,7 @@
   - Namespaces: The example namespaces are now backend-specific. Instead of `rook-system` and `rook`, you will see `rook-ceph-system` and `rook-ceph`.
   - Volume plugins: The dynamic provisioner and flex driver are now based on `ceph.rook.io` instead of `rook.io`
 - Ceph container images now use CentOS 7 as a base
+- If your `monCount` is `0`, it is now defaulted to `3`.
```

Comment:
@travisn Should this be mentioned?
Previous behavior was to use 1 mon for 1 node environments and three for three or more nodes.

Reply:
It will be the same default in this case from 0.7 to 0.8, right? maybe we don't need to mention it

Reply:
Depends on the environment, previously the default was 1 mon for a 1-2 node cluster and 3 mons for a 3 and more cluster.
So there is a change.

Reply:
it's just a change from a previous master build, but not a change from 0.7, right? I don't think we need to mention if it's just a master change and it's not critical like this

Reply:
Oh right, just checked the `v0.7` code. I'll remove it.